ksm.go

```diff
@@ -129,8 +132,53 @@ func isKSMAvailable(ksmDir string) error {
 	return nil
 }
 
+func systemPages() (int64, error) {
+	// We're going to parse meminfo
+	f, err := os.Open("/proc/meminfo")
+	if err != nil {
+		return -1, err
+	}
+	defer f.Close()
+
+	scan := bufio.NewScanner(f)
+	for scan.Scan() {
+		line := scan.Text()
+
+		// We only care about MemTotal
+		if !strings.HasPrefix(line, "MemTotal:") {
```

Comment:
Rather that looking at mem total, we should look at anonymous pages. That is what KSM tracks anyway. Also we should update this post creation of each VM as the anon pages changes. That way we are actually tracking what is happening on the system the total memory size of the system, which can be very large.

Reply:
Good point. We're now only tracking anon pages.